.github/actions/kind/action.yml

```diff
@@ -56,3 +62,13 @@ runs:
         curl https://raw.githubusercontent.com/kubernetes/ingress-nginx/"${VERSION}"/deploy/static/provider/kind/deploy.yaml | sed "s/--publish-status-address=localhost/--report-node-internal-ip-address\\n        - --status-update-interval=10/g" | kubectl apply -f -
         kubectl annotate ingressclass nginx "ingressclass.kubernetes.io/is-default-class=true"
         kubectl -n ingress-nginx wait --timeout=300s --for=condition=Available deployments --all
+
+    - name: Add ${{ inputs.kind-node-hostname }} host to machine hosts
+      shell: bash
+      run: echo "127.0.0.1 ${{ inputs.kind-node-hostname }}" | sudo tee -a /etc/hosts
```

Comment:
Is this really needed?

Reply:
Currently no, though in future I plan to store Ray job logs in SDK test, for that I will need it.

Reply:
OK sounds good.